core/server/master/src/main/java/alluxio/master/meta/MetaMasterInfo.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.meta;
+
+import com.google.common.base.Objects;
+import com.google.common.base.Preconditions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.concurrent.NotThreadSafe;
+
+/**
+ * Metadata for an Alluxio standby masters.
+ * This class is not thread safe, so external locking is required.
+ */
+@NotThreadSafe
+public final class MetaMasterInfo {
```

Comment:
Since this is info about a master, could we call it just `MasterInfo`?

Reply:
better name! Changed!